docs/manual/cluster-admin/how-to-set-up-docker-image-cache.md

```diff
@@ -0,0 +1,104 @@
+* `docker_cache_htpasswd`: htpasswd auth info with base64 encoded, should be use with ssl when docker-cache cache some private registry as an access control method.
+
+### `config.yaml` example with azure
+
+``` yaml
+user: forexample
+password: forexample
+docker_image_tag: v1.5.0
+
+# Optional
+
+#######################################################################
+#                    OpenPAI Customized Settings                      #
+#######################################################################
+# enable_hived_scheduler: true
+enable_docker_cache: true
+docker_cache_storage_backend: "azure"
+docker_cache_azure_account_name: "forexample"
+docker_cache_azure_account_key: "forexample"
+# docker_cache_azure_container_name: "dockerregistry"
+# docker_cache_fs_mount_path: "/var/lib/registry"
+# docker_cache_remote_url: "https://registry-1.docker.io"
+# docker_cache_htpasswd: "" 
+# enable_marketplace: "true"
+
+# ...
+
+```
+
+Make sure the setting of `enable_docker_cache` was `"true"` (include the quotation marks), and finish the [installation](./installation-guide.md), the marketplace will be set up.
+
+### `config.yaml` example with filesystem
+
+``` yaml
+user: forexample
+password: forexample
+docker_image_tag: v1.5.0
+
+# Optional
+
+#######################################################################
+#                    OpenPAI Customized Settings                      #
+#######################################################################
+# enable_hived_scheduler: true
+enable_docker_cache: true
+docker_cache_storage_backend: "filesystem"
+# docker_cache_azure_account_name: ""
+# docker_cache_azure_account_key: ""
+# docker_cache_azure_container_name: "dockerregistry"
+docker_cache_fs_mount_path: "/var/lib/registry"
+# docker_cache_remote_url: "https://registry-1.docker.io"
+# docker_cache_htpasswd: "" 
+# enable_marketplace: "true"
+
+# ...
+
+```
+
+Make sure the setting of `enable_docker_cache` was `"true"` (include the quotation marks), and finish the [installation](./installation-guide.md), the marketplace will be set up.
+
+### htpasswd explained
+
+The *htpasswd* authentication backed allows you to configure basic authentication using an [Apache htpasswd file](https://httpd.apache.org/docs/2.4/programs/htpasswd.html).
+The only supported password format is bcrypt. Entries with other hash types are ignored. The htpasswd file is loaded once, at startup. If the file is invalid, the registry will display an error and will not start. 
+
+In docker-cache service, we use htpasswd info as k8s secret, which means `docker_cache_htpasswd` need base64 encoded htpasswd file content.
+
+## Set Up Docker Image Cache for deployed cluster
+
+For cluster already deployed situation, enable docker-cache service is no need to totally re-install. The suggested way is modify config.yaml as in installation, and use following command to do upgrade.
```

Comment:
For those who already deployed the cluster, there is no need to re-install the cluster totally to enable docker-cache service. The suggested way is to modify `config.yaml`, and use the following commands to upgrade.